src/Symfony/Component/Messenger/Middleware/HandleMessageMiddleware.php

```diff
@@ -75,7 +75,7 @@ public function handle(Envelope $envelope, StackInterface $stack): Envelope
                 throw new NoHandlerForMessageException(sprintf('No handler for message "%s".', $context['class']));
             }
 
-            $this->logger->info('No handler for message "{class}"', $context);
+            $this->logger->info('No handler for message {class}', $context);
```

Comment:
Removed quotes for consistency. They are not useful in logs because in cli they are colored due to the context and the class is a very fixed string anyway without whitespace.